docs/rules/define-props-declaration.md

```diff
@@ -39,15 +41,23 @@ const props = defineProps({
 ## :wrench: Options
 
 ```json
-  "vue/define-props-declaration": ["error", "type-based" | "runtime"]
+{
+  "vue/define-props-declaration": ["error",
+    "type-based" | "runtime",
+    {
+      "autoFixToSeparateInterface": false
```

Comment:
I don't think this is a good option, if there is more than one way to fix it, I think it should be implemented as a suggested feature.
Could you please use the suggestions API instead of auto-fix?

https://eslint.org/docs/latest/extend/custom-rules#providing-suggestions

Reply:
I'll just remove this option in that case. I don't want to overcomplicate it. 
Also, preferring inline types vs define separate one can be probably handled by a different linter rulse.

Reply:
Please also remove it from the docs again.